index.bzl

```diff
@@ -39,6 +40,7 @@ pkg_npm = _pkg_npm
 npm_package_bin = _npm_bin
 pkg_web = _pkg_web
 copy_to_bin = _copy_to_bin
+params_file = _params_file
 # ANY RULES ADDED HERE SHOULD BE DOCUMENTED, see index.for_docs.bzl
```

Comment:
add to index.for_docs.bzl so it gets stardoc

Reply:
done. thanks